ui/app/templates/partials/transit-form-show.hbs

```diff
@@ -52,16 +52,16 @@
           <div class="td is-borderless">
             {{version}}
             {{#if (coerce-eq key.minDecryptionVersion version)}}
-            <p class="help has-text-grey">(current minimum decryption version)</p>
+              <p class="help has-text-grey">(current minimum decryption version)</p>
             {{/if}}
           </div>
         </div>
         <div class="column is-half td">
           <div class="td is-borderless">
-            {{moment-format (unix creationTimestamp) 'MMM DD, YYYY hh:mm:ss A'}}
-            <br/>
+            {{date-format (creationTimestamp) 'MMM DD, YYYY hh:mm:ss A'}}
```

Comment:
I think you want this w/o the parens - with the parens, ember will try to find a helper called `creationTimestamp` and that will error. Without them, it'll look up the value of `creationTimestamp` in the current rendering context.